pkg/controller/account/ec2.go

```diff
@@ -258,3 +323,266 @@ func ListEC2InstanceStatus(reqLogger logr.Logger, client awsclient.Client) (*ec2
 
 	return result, nil
 }
+
+// getDesiredVCPUValue retrieves the desired quota information from the operator configmap and converts it to a float64
+func (r *ReconcileAccount) getDesiredVCPUValue(reqLogger logr.Logger) (float64, error) {
+	var err error
+	var vCPUQuota float64
+
+	configMap, err := utils.GetOperatorConfigMap(r.Client)
+	v, ok := configMap.Data["quota.vcpu"]
+	if !ok {
+		err = awsv1alpha1.ErrInvalidConfigMap
+	}
+	if err != nil {
+		reqLogger.Error(err, "failed getting desired vCPU quota from configmap data")
+		return vCPUQuota, err
+	}
+
+	vCPUQuota, err = strconv.ParseFloat(v, 64)
```

Comment:
Why parse float and not parse int?

Reply:
The vCPUQuota request below requires a float64 type, so I parsed it as such here to avoid parsing from string to int and then to float later.

Reply:
ah got it. Also, cmon AWS, why do you need a float64 for a whole number???